src/System.Management.Automation/engine/remoting/commands/EnterPSHostProcessCommand.cs

```diff
@@ -535,9 +544,13 @@ internal static IReadOnlyCollection<PSHostProcessInfo> GetAppDomainNamesFromProc
                 if (startIndex > -1)
                 {
                     // This is a PowerShell named pipe.  Parse the process Id, AppDomain name, and process name.
+#if !UNIX
                     int pStartTimeIndex = namedPipe.IndexOf(".", startIndex, StringComparison.OrdinalIgnoreCase);
```

Comment:
We should replace `"."` string with char `'.'
```suggestion
                    int pStartTimeIndex = namedPipe.IndexOf('.', startIndex, StringComparison.OrdinalIgnoreCase);
```